src/Symfony/Component/Yaml/Parser.php

```diff
@@ -101,6 +115,16 @@ public function parse($value, $flags = 0)
             mb_internal_encoding('UTF-8');
         }
 
+        Inline::$tagResolver = $this->tagResolver;
+        try {
+            return $this->doParse($value, $flags);
+        } finally {
+            Inline::$tagResolver = null;
```

Comment:
@nicolas-grekas this is only done for bc (`Inline` wasn't internal in 3.0) but it must not be used in userland so I think it's fine to always remove the previous value.

@stof it is following [this](https://github.com/symfony/yaml/blob/master/Parser.php#L159). I think it would be better to instantiate `Inline` and pass the `TagResolver` as a constructor argument, wdyt?